docs/conf.py

```diff
@@ -76,9 +77,9 @@
 # the built documents.
 #
 # The short X.Y version.
-version = ""
+version = clisops.__version__
 # The full version, including alpha/beta/rc tags.
-release = "0.3.1"
+release = clisops.__version__
 
```

Comment:
This could be managed by `bumpversion`, preventing us from needing to load the `clisops` library when we build the docs. Are we certain this isn't going to lead to memory limit issues on RtD?

Reply:
Good point - I missed this one. Updated this to use `bumpversion` and removed the import.